src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -0,0 +1,287 @@
+          {options}
+        </select>
+      </div>
+    );
+  }
+
+  _createInputFilter(filter: DataTableFilter, index: number) {
+    return (
+      <div
+        id={this._calculateFilterId(filter.name)}
+        // Todo
+        class={`
+          ${FORM_CLASSES.FORM_ITEM}
+          ${index !== 0 && !filter.advanced ? 'chi-data-table-filter__input -ml--2' : ''}`}>
+        <input
+          aria-label={`Filter by ${filter.label || filter.name}`}
+          class={`${INPUT_CLASSES.INPUT} -lg`}
+          data-filter={filter.name}
+          onChange={(ev: Event) => this._changeFormElementFilter(ev, 'input')}
+          placeholder={filter.placeholder || null}
+          type="text"
+          value={filter.value || ''}
+        />
+      </div>
+    );
+  }
+
+  _createTextareaFilter(filter: DataTableFilter, index: number) {
+    return (
+      // Todo
+      <div
+        class={`${FORM_CLASSES.FORM_ITEM} ${
+          index !== 0 && !filter.advanced ? 'chi-data-table-filter__input -ml--2' : ''
+        }`}>
+        <textarea
+          id={this._calculateFilterId(filter.name)}
+          aria-label={`Filter by ${filter.label || filter.name}`}
+          data-filter={filter.name}
+          class={`${INPUT_CLASSES.INPUT} -lg`}
+          placeholder={filter.placeholder || null}
+          onChange={(ev: Event) => this._changeFormElementFilter(ev, 'textarea')}
+          value={filter.value || ''}
+        />
+      </div>
+    );
+  }
+
+  _createCheckboxFilter(filter: DataTableFilter, index: number) {
+    return (
+      <div
+        class={`
+          ${FORM_CLASSES.FORM_ITEM}
+          ${index !== 0 && !filter.advanced ? '-ml--2' : ''}
+          ${UTILITY_CLASSES.DISPLAY.FLEX}
+          ${UTILITY_CLASSES.JUSTIFY.CENTER}`}>
+        <div class={`${CHECKBOX_CLASSES.checkbox} ${UTILITY_CLASSES.ALIGN_SELF.CENTER}`}>
+          <input
+            id={`toolbar-filter-checkbox__${filter.name}`}
+            aria-label={`Filter by ${filter.label || filter.name}`}
+            data-filter={filter.name}
+            type="checkbox"
+            class={CHECKBOX_CLASSES.INPUT}
+            checked={filter.checked}
+            onChange={(ev: Event) => this._changeFormElementFilter(ev, 'checkbox')}
+          />
+          <label for={`toolbar-filter-checkbox__${filter.name}`} class={CHECKBOX_CLASSES.LABEL}>
+            {filter.name}
+          </label>
+        </div>
+      </div>
+    );
+  }
+
+  _emitAdvancedFiltersChange() {
+    this.$emit(DATA_TABLE_EVENTS.ADVANCED_FILTERS_CHANGE);
+  }
+
+  _applyAdvancedFiltersChange() {
+    this._emitAdvancedFiltersChange();
+    this._toggleAdvancedFiltersPopover();
+    this._advancedFiltersAccordion.collapseAll();
+  }
+
+  _toggleAdvancedFiltersPopover() {
+    if (this._advancedFilterPopoverId) {
+      const popover = document.getElementById(this._advancedFilterPopoverId);
+
+      if (popover) {
+        // eslint-disable-next-line
+        // @ts-ignore
```

Comment:
Needs to be ignored as `.toggle()` is been provided once the web component is hydrated.